zeebe/test-util/src/main/java/io/camunda/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -97,11 +97,11 @@ public class CompactRecordLogger {
           entry("COMPLETION_DENIED", "COMP_DENIED"),
           entry("UPDATE_DENIED", "UPDT_DENIED"),
           entry("SEQUENCE_FLOW_TAKEN", "SQ_FLW_TKN"),
+          entry("AD_HOC_SUB_PROCESS_INSTRUCTION", "AHSP_INST"),
```

Comment:
Moved this up here as otherwise it would replace `PROCESS` with `PROC` first, so the mapping would need to be for `AD_HOC_SUB_PROC_INSTRUCTION` which is pretty misleading.